docs/core/howto/ssl.rst

```diff
@@ -204,10 +199,40 @@ For example,
     options = CertificateOptions(..., dhParameters=dhParams)
 
 Another part of the TLS protocol which ``CertificateOptions`` can control is the version of the TLS or SSL protocol used.
-This is often called the context's "method".
+There are two ways to configure it, either by configuring a list of protocols, or by the legacy way of setting the internal context's "method".
+By default, Twisted will configure it to use TLSv1.0 or later and disable the insecure SSLv3 protocol, but this manual control over protocols can be helpful if you need to support legacy SSLv3 systems, or you wish to restrict it down to just the strongest of the TLS versions.
+
+You can give ``CertificateOptions`` a range of TLS versions you wish it to negotiate with the ``tlsProtocols`` argument in the initializer:
+
+.. code-block:: python
+
+    from twisted.internet.ssl import (CertificateOptions, TLSVersion,
+                                      TLSVersionRange)
+    options = CertificateOptions(
+        ...,
+        tlsProtocols=TLSVersionRange(TLSVersion.TLSv1_0,
+                                     TLSVersion.TLSv1_2))
+
```

Comment:
I'm inclined to say that we should just give `CertificateOptions` two arguments; `requireAtLeastVersion=` and, say, `artificiallyLowerProtocolMaximum=`, to mirror their functional purpose.  If we just use symmetric `minimumVersion`/`maximumVersion` argument names, it makes it sound like the range is just arbitrary, but in fact opinionated lower levels and implementation restrictions might decide to ignore our instructions anyway since these have specific security implications; the minimum and maximum can then have different documentation that explains the situation in which each might be ignored.